vue/src/components/LanguageSelector.vue

```diff
@@ -0,0 +1,198 @@
+<template>
```

Comment:
As far as i can see this component is not used anywhere which means updateUserPayload is never called. From what i can see the language selecter is directly in the footer?